README.md

```diff
@@ -1,10 +1,32 @@
 # Ambianic-subscriptions
+
 ## Repo for ambianic premium subscription management
 
 <p> This repo hosts [Netlify functions](https://www.netlify.com/products/functions/) that handle Ambianic premium user subscriptions. This is code that cannot run in the browser PWA, because it needs access keys to subscription and payment gateway services (Stripe) shared between all app users. If the code runs in the PWA, users will be able to see and manipulate each other's premium subscription data </p>
 
-
 The image below shows the architecture of these serverless functions in connection with the Ambianic [PWA](https://github.com/ambianic/ambianic-ui/).
 ![serverless-functions-architecture](./assets/serverless-architecture.png)
 
- 
+## Running Locally
+
+The serverless functions within this project are managed using [netlify-dev](https://www.netlify.com/products/dev/). Start the functions emulator from the `netlify` directory using `yarn netlify-dev` to listen for requests to any of created functions on port `5050`.
+
+Credentials within this project are managed using environment variables and [GitHub Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets). To run the functions here locally using the `netlify dev` command, create a `.env` file in the root directory with the following values;
+
+```
+STRIPE_TEST_KEY=STRIPE_KEY
+
+# The product ID which users are subscribed to.
+EMAIL_PRODUCT_ID=STRIPE_KEY
+```
+
+## CI / CD Pipeline
+
+The three currently available serverless function endpoints are all tested using [Postman Tests](https://www.postman.com/automated-testing/). The API documentation explaining the endpoints can be found in the GitHub pages for this repository [here](https://vickywane.github.io/ambianic-subscriptions.github.io/). An exported file of the postman collection in json format is available at `./tests/postman/ambianic-functions-collection.postman_collection.json`.
```

Comment:
@vickywane Is this statement is incorrect after the conversion to Prism?

Reply:
Yes, it is correct. 

Prism is only being used to provide a local mock server that returns data in a defined format.

The collection's documentation / definition is still managed with Postman and for now, we still have Postman Tests being used,